_posts/2017-10-06-cockpit-152.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Cockpit 152
+author: pitti
+date: 2017-10-06 09:00
+tags: cockpit linux
+slug: cockpit-152
+category: release
+summary: Cockpit with new Applications page and automatic software updates
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 152.
+
+### Add Applications page
+
+This version introduces the "Applications" tool page. This is an extension
+point for third-party software to add themselves to the Cockpit interface, by
+shipping [AppStream](https://www.freedesktop.org/wiki/Distributions/AppStream/)
+metadata that contains a Cockpit manifest.
```

Comment:
Strictly speaking, the AppStream metadata doesn't contain the Cockpit manifest.  "by shipping appropriate AppStream metadata".

Reply:
Indeed, the AppStream data just contains the `<launchable type="cockpit-manifest">` tag, not the manifest itself. Adjusted, thanks!